rfcs/formats/communication/relay.md

```diff
@@ -12,3 +12,51 @@
 
 ## Summary
 
+
+## NetworkQuery
+
+This message is created by client application to encapsulate the query for the relay.
+
+```protobuf
+message NetworkQuery {
+  repeated string policy = 1;
+  string address = 2;
+  string requesting_relay = 3;
+  string requesting_network = 4;
+  string certificate = 5;
+  string requestor_signature = 6;
+  string nonce = 7;
+  string requesting_org = 8;
+}
+```
+
+`policy` field specifies the [verification policy](../policies/proofs-verification) to be used for this query.
+`address` stores the view address. More on addressing [here](../views/addressing).
+Rest fields are self explanatory.
```

Comment:
I think we should describe each field:
- signature is used to ensure that the query has not been tampered with
- certificate is used to authenticate the signer and optionally also for access control
- nonce is used to avoid replay attacks by entities masquerading as an application
- (also mention what the remaining fields are used for)